### PR TITLE
add DOI to README

### LINKED ISSUE #23: Final stuff?
@martinfleis Any last stuff you can think of for tomorrow?

### PR DESCRIPTION
**Pull request by jGaboardi**
Resolves #23.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Understanding the structure of cities through the lens of data
 
-### SDSS 2022 Workshop
+### SDSS 2022 Workshop [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7109016.svg)](https://doi.org/10.5281/zenodo.7109016)
```
Comment (martinfleis) [inline README.md]: I would put it under the title, it is not really aligned if it is next.

File: demo-notebook.ipynb [hunk 2]
```diff
@@ -46,28 +46,15 @@
...
-       " <iframe height=\"750\" width=\"100%\" src=\"https://martinfleis.github.io/sdss22-workshop/slides/intro.html\", style=\"border:0\"></iframe>\n"
-      ],
-      "text/plain": [
-       "<IPython.core.display.HTML object>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
```
Comment (martinfleis) [inline demo-notebook.ipynb]: Can we keep this cell executed so the presentation is visible?

File: README.md [hunk 3]
```diff
@@ -17,9 +19,7 @@ Martin & James will walk you through the fundamentals of analysis of the structu
 
 ### Run online
 
-Open in an interactive in-browser environment:
-
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/martinfleis/sdss22-workshop/HEAD?labpath=demo-notebook.ipynb)
+Open in an interactive in-browser environment: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/martinfleis/sdss22-workshop/HEAD?labpath=demo-notebook.ipynb)
```
Comment (jGaboardi) [inline README.md]: What about these Binder buttons? I changed then to in-line. Think it's better to move them back?

Comment (martinfleis) [reply README.md]: I like it more on a separate line as the buttons are not aligned with the text due to different sizes but that is just a personal preference and doesn't really matter.

**Review by martinfleis — APPROVED** [review]

**Pull request merged**